schemas/did.js

```diff
@@ -18,6 +18,9 @@ const schema = {
   description: 'A decentralized identifier.',
   type: 'string',
   pattern,
+  minLength: cfg.environment === 'test' ? 17 : 12,
```

Comment:
```suggestion
  minLength: prefix.length + 6,
```

Reply:
I see this happens in multiple places. If these are all based off of the prefix, just use `prefix.length` and add to the base value everywhere.